components/climate/pid.rst

```diff
@@ -28,12 +28,17 @@ but there's a nice article explaining the function principle `here <https://blog
           kp: 0.49460
           ki: 0.00487
           kd: 12.56301
-
+          output_samples: 5      # smooth the output over 5 samples
+          derivative_samples: 5  # smooth the derivative value over 10 samples
+        deadband_parameters:
+          threshold_high: 0.5°C       # deadband within +0.5°C of target_temperature
+          threshold_low: -1.0°C       # deadband within -1.0°C of target_temperature
```

Comment:
Please set the values for the output_samples etc to 0 or remove them from the block. Reason is many people just copy this into their config (without looking into what each option does) and so the sample config should be close to the default in order to avoid undesired behavior.